infrastructure/conf/nginx-conf-local/default

```diff
@@ -153,6 +153,13 @@ server {
         include fastcgi_params;
         fastcgi_param SCRIPT_FILENAME $document_root/api/public/index.php;
     }
+
+    # simulate being off the VPN and the firewall redirecting to the restricted page
+    # location = /admin/graphql {
```

Comment:
Shouldn't this be
```suggestion
    # location = /admin {
```
?

Reply:
Oh yeah, that makes sense

Reply:
Oh wait, the equals is an exact match.  This probably won't work anymore.

Reply:
Ah, you're right

Reply:
I think this might mirror what Prod is doing? When I use this, I see the same behaviour as on prod. Unfortunately, the error toast also seems to have stopped coming up...

```
    location ~ ^/(en|fr)?/admin {
        absolute_redirect on;
        return 301 http://localhost:8000/restricted.html;
    }
```

Reply:
OK, I reworked this block to behave more like the prod one.
[reword firewall simulator](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12226/commits/3296a1247ebfa5cd6ce613e8b157cb6681390d29)

Reply:
Since the regex version has a lower importance than an exact match I added a comment that you'll need to comment out the main rule for /admin/graphql since it will just silently override the firewall simulation rule otherwise.